db/seeds.rb

```diff
@@ -101,3 +101,15 @@
   rating: 2,
   content: "Not very funny..."
   )
+
+QueueItem.create!(
+  user_id: alice.id,
+  video_id: the_office.id,
+  position: 1
+)
+
+QueueItem.create!(
+  user_id: alice.id,
+  video_id: the_amazing_race.id,
```

Comment:
Instead of assigning id's, you can just do `video: the_amazing_race` - its more succinct without all those ids. Same for user and above lines.